core/src/main/java/hudson/model/Node.java

```diff
@@ -214,8 +217,13 @@ public final VirtualChannel getChannel() {
 
     /**
      * Creates a new {@link Computer} object that acts as the UI peer of this {@link Node}.
+     * 
      * Nobody but {@link Jenkins#updateComputerList()} should call this method.
+     * @return Created instance of the computer.
+     *         Can be {@code null} if the {@link Node} implementation does not support it (e.g. {@link Cloud} agent).
      */
+    @CheckForNull
+    @Restricted(NoExternalUse.class)
```

Comment:
@stephenc @jglick My intention was to reflect the `Nobody but {@link Jenkins#updateComputerList()} should call this method.` comment in Javadoc. If you know a better annotation for it, just let me know. Not sure if I can dedicate time to writing another restriction if it is missing, so I will just remove it in such case